### PR TITLE
Feature/dictionary extension

### PR DESCRIPTION
**Pull request by siwuagain**
Hello, 

I finished implementing the first version of the synonym and antonym command. Since the logic is identical, I used two mutual functions: one to get all the synonyms/antonyms of the word and one to display the results in a discord embed.  This is what an example embed with multiple pages look like: 
image

In the case where there are no synonym(s)/antonym(s) for the word or the word is misspelled, the bot sends a message like so: 
image

I have two alternatives in case you'd like to me touch up on the display. I could display the message in an embed rather than a message. I could also make it so that the message indicates whether no results were found because the thesaurus library couldn't find any or rather the word just doesn't exist (misspelled possibly). Let me know your input on these thoughts and if you have anything to add. For now, no pertinent dictionary extension commands come to mind, so I will be moving on to work on other features, like the economy module, unless you have any possible features to discuss or suggestions for the syn/ant commands.

### CODE REVIEW

File: cogs/fun.py [hunk 1]
```diff
@@ -95,9 +96,59 @@ async def dictionary(self, ctx: commands.Context, *, word: str):
...
+            else:
+                fields.append(field)
+                field = []
+                max_char = 1024
+        for field in fields:
+            page = discord.Embed(title=f"Zen | Thesaurus")
+            page.add_field(name="Word", value=f"{word.title()}", inline=False)
+            page.add_field(name=f"{title}", value=f"{', '.join(field)}", inline=False)
+            embeds.append(page)     
+        return embeds
```
Comment (deemo-o) [inline cogs/fun.py]: Could move these functions above all the commands functions to make this file look cleaner.

**Review by deemo-o — APPROVED** [review]
I do think that displaying the error message in an embed would be a lot better since we are mostly using embeds in every other modules.

Other than that, great work! Everything looks good to me.

File: cogs/fun.py (hunk 1 above)
Comment (siwuagain) [reply cogs/fun.py]: Will do!

**Review by Kenny-star — APPROVED** [review]
Good Job!

**Pull request merged**